crates/wasmtime/src/module.rs

```diff
@@ -1117,6 +1136,17 @@ impl Drop for ModuleInner {
     }
 }
 
+/// Describes the location of an export in a module.
+#[derive(Copy, Clone)]
+pub struct ModuleExport {
+    /// The module that this export is defined in.
+    pub module: CompiledModuleId,
+    /// A raw index into the wasm module.
+    pub entity: EntityIndex,
+    /// The index of the export name.
+    pub export_name_index: usize,
```

Comment:
Mind making these private and/or `pub(crate)`? Ideally users shouldn't have to actually look at these fields.